app/assets/stylesheets/views/_search.scss

```diff
@@ -135,8 +135,9 @@ main.search {
       }
 
       mark {
-        background-color: $focus-colour;
+        background-color: inherit;
```

Comment:
@robinwhittleton not if the mark will be inside an `<a>` tag, and may have a `:focus` state with a background colour.